### ISSUE TITLE
Calendar issue

### ISSUE THREAD

**Issue by khakha-sys** [usnistgov/NEMO#264]
Reservations are not visible on my calendar ,but when i search my user through search for a specific user and then type my username i can see all the entries there.

**Comment by rptmat57** [comment]
do you have access to NEMO's logs?
otherwise can you check the web browser's console for any error messages?

**Comment by khakha-sys** [comment]
Yes I have access to logs ,There is nothing wrong showing in logs for calendar or any restriction. I can see the reservation in personal when it is created for a tool but i cant see it when i click on the tool itself.

**Comment by rptmat57** [comment]
hmm, that's odd. usually when that happens there is some kind of error in the web browser's console (on Chrome: Menu -> More tools -> developer tools)
Which version of NEMO are you on and which browser are you using?

**Comment by rptmat57** [comment]
just to be clear, you can see the reservation in "Personal schedule", and also in "specific user" when typing you name, but not in the tool calendar itself?
did you make sure to clear all selections, and also try in incognito mode?
you can also try to run `localStorage.clear()` in the console and try again

**Comment by khakha-sys** [comment]
Sorry for the delayed response , 
I deleted the tools and created new tools now the issue is only admin user which was first created in nemo can create reservations , and the other user which was created through web portal can see it as well but as soon the other user try to create an entry in the same calendar everything disappears from both users calendar but from admin portal we can see all reservations are there. 
[Recording 2025-01-09 153202.zip](https://github.com/user-attachments/files/18367133/Recording.2025-01-09.153202.zip)
![Screenshot 2025-01-09 153228](https://github.com/user-attachments/assets/3cb2eeae-4f9a-4502-a6b6-795df671dc33)
I can give you remote session if possible for you that will help us a lot to solve the issue. Thank you in advance..

**Comment by rptmat57** [comment]
very strange. send me an email at mathieu.rampant@atlantislabs.io and I will see if I can take a look at your instance.
